### PR TITLE
Enable the Rails/FindBy cop in Rubocop && Refactor UpdateinfoCounter#find_or_create

### PR DESCRIPTION
**Pull request by Ana06**
Enable the `Rails/FindBy` cop in Rubocop: Note that `where(..).first` is not exactly the same as `find_by(...)` as they order records differently, so the "first" record can be different. Because of this reason the Rubocop auto-correct option doesn't work and all the changes have been done manually.

I've also refactored the `UpdateinfoCounter#find_or_create` method. :bowtie:

### CODE REVIEW

File: src/api/.rubocop_todo.yml [hunk 1]
```diff
@@ -216,12 +216,10 @@ Rails/Exit:
     - 'config/deploy.rb'
     - 'lib/memory_dumper.rb'
 
-# Offense count: 25
 # Cop supports --auto-correct.
 # Configuration parameters: Include.
-# Include: app/models/**/*.rb
 Rails/FindBy:
-  Enabled: false
+  Enabled: true
```
Comment (hennevogel) [inline src/api/.rubocop_todo.yml]: Please enable it over at .rubocop.yml and remove it here

**Comment by Ana06** [comment]
@hennevogel can we merge this or is there something else to change?

File: src/api/.rubocop.yml [hunk 2]
```diff
@@ -7,6 +7,14 @@ AllCops:
     - 'lib/templates/**/*'
     - 'vendor/bundle/**/*'
 
+# Cop supports --auto-correct.
+# Configuration parameters: Include.
+# auto-correct only works for where(...).take but not for where(...).first, as 
+# where(...).first and find_by can return different results (they order records
+# differently, so the 'first' record can be different).
```
Comment (bgeuken) [inline src/api/.rubocop.yml]: Let's remove the comments. We settled for find_by and won't them anymore

**Comment by bgeuken** [comment]
LGTM, besides the superfluous comment

**Comment by Ana06** [comment]
[Enable the Rails/FindBy cop in Rubocop](https://trello.com/c/OzXZWAoA/77-enable-the-rails-findby-cop-in-rubocop)

**Pull request merged**